src/components/menu/menu-trigger/menu-trigger.directive.ts

```diff
@@ -33,7 +34,12 @@ export class MenuTriggerDirective implements OnInit, OnDestroy {
     private _portal: TemplatePortal;
 
     /** Store the reference to the overlay */
-    private _overlayRef: OverlayRef;
+    private _overlayRef?: OverlayRef;
+
+    /** Get the aria controls for accessibility */
+    get ariaControls(): string | null {
+        return this.menu?.isMenuOpen ? this._overlayRef?.overlayElement.id : null;
```

Comment:
I don't think this is right. According to this reference, the `aria-controls` attribute should refer to the element with `role="menu"`:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/menu-button/menu-button-links.html

So `ux-menu` should accept an `id` input and derive an id for the `role="menu"` element, e.g. `[attr.id]="id + '-menu'"`. `aria-controls` will contains the same value. If the developer has not provided an `id` for `ux-menu`, one should be generated. Check the ux-select component for an example of id generation for child elements.